src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -3051,6 +3051,62 @@ public LazyObjectCreationExpression(IMethodSymbol constructor, Lazy<ImmutableArr
     }
 
     /// <summary>
+    /// Represents a C# or VB new/New anonymous object creation expression.
+    /// </summary>
+    internal abstract partial class BaseAnonymousObjectCreationExpression : Operation, IAnonymousObjectCreationExpression
+    {
+        protected BaseAnonymousObjectCreationExpression(bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+                    base(OperationKind.AnonymousObjectCreationExpression, isInvalid, syntax, type, constantValue)
+        {
+        }
+        /// <summary>
+        /// Explicitly-specified member initializers.
+        /// </summary>
+        public abstract ImmutableArray<IOperation> Initializers { get; }
+        public override void Accept(OperationVisitor visitor)
+        {
+            visitor.VisitAnonymousObjectCreationExpression(this);
+        }
+        public override TResult Accept<TArgument, TResult>(OperationVisitor<TArgument, TResult> visitor, TArgument argument)
+        {
+            return visitor.VisitAnonymousObjectCreationExpression(this, argument);
+        }
+    }
+
+    /// <summary>
+    /// Represents a C# or VB new/New anonymous object creation expression.
+    /// </summary>
+    internal sealed partial class AnonymousObjectCreationExpression : BaseAnonymousObjectCreationExpression, IAnonymousObjectCreationExpression
+    {
+        public AnonymousObjectCreationExpression(ImmutableArray<IOperation> initializers, bool isInvalid, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+            base(isInvalid, syntax, type, constantValue)
+        {
+            Initializers = initializers;
+        }
+        /// <summary>
+        /// Explicitly-specified member initializers.
+        /// </summary>
+        public override ImmutableArray<IOperation> Initializers { get; }
```

Comment:
Should the Initializers use a more specific type/interface? I see this was considered in the design notes, but all the tests show IAssignmentExpression goes there. Is it possible to get anything else?